### PR TITLE
first release of dotnet SDK extension 6.0rc1

### PR DESCRIPTION
**Pull request by patriziobruno**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64", "arm", "aarch64"],
```
Comment (nedrichards) [inline flathub.json:2]: arm isn't supported by fdo 21.08 so it's unnecessary here.

**Comment by nedrichards** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet6

**Comment by patriziobruno** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet6

**Comment by patriziobruno** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet6

**Pull request closed without merging**